### ISSUE TITLE
[Annotation] drawing tool acts weird if you double click and click to drag  on the image

### ISSUE THREAD

**Issue by ianb** [mozilla-services/screenshots#3986]
If I use the drawing tool for a while (just scribbling) I eventually get into a state where the tool will only draw a short distance and then stops (probably one mousemove). The mouse cursor also changes to the default pointer cursor.

I don't know exactly what triggers it. I tried right-clicking, and other things, but none of them put it into this particular state. Given this I'm guessing its a timing issue. But after 30 seconds of scribbling I can usually get it back into this state.

There are no errors in the console. Switching tools does not make the bug go away.

**Comment by johngruen** [comment]
@SoftVision-CosminMuntean can you see how long it takes you to repro this?

**Comment by SoftVision-CosminMuntean** [comment]
I haven't managed to reproduce this issue.  I was able to draw continuously without releasing the mouse more then 1 minute.  
I have tested it using Screenshots 28.0.0 dev version on latest Nightly 60.0a1 on Mac 10.12, Windows 7 x64 and Arch Linux.
I have notice that the mouse cursor is changing on Mac only when I hover the margins of the browser. But if I click again on any of the drawing tools the cursor is correctly displayed.

@ianb did you encountered this issue on Mac using the touchpad?

**Comment by SoftVision-CosminMuntean** [comment]
After further investigations I finally managed to reproduce it using the following steps to reproduce:

[Steps to reproduce]:
1. Open Firefox browser with the latest Screenshots 28.0.0 dev version installed.
2. Navigate to a saved shot link and click the "Edit this image" button.
3. Double click on the image without moving the cursor and click to drag in order to draw on the image.
4. Observe the behavior.

[Actual results]:
- The draw stops while dragging the mouse cursor.

I have noticed different behaviors on Mac 10.12, Windows 10, 7 x64 and Ubuntu 14.0.4:
- on Mac, the mouse cursor is changed in the default one.
- on Windows, the mouse cursor is changed in an unavailable cursor while dragging. Like dragging a file into a restricted area.
- on Ubuntu, while dragging on the image, a file icon is displayed in the bottom right corner of the cursor.

Here is a screen recording of the issue on Windows:
![color picker -draw issue](https://user-images.githubusercontent.com/19226534/35691865-532929dc-0782-11e8-8747-82f09463aa97.gif)
